pennylane_lightning/core/src/simulators/lightning_gpu/measurements/MeasurementsGPU.hpp

```diff
@@ -259,10 +259,10 @@ class Measurements final
      * @return auto Expectation value.
      */
     template <class index_type>
-    auto expval(const index_type *csrOffsets_ptr,
-                const index_type csrOffsets_size, const index_type *columns_ptr,
+    auto expval(const index_type *csrOffsets_ptr, const int64_t csrOffsets_size,
+                const index_type *columns_ptr,
                 const std::complex<PrecisionT> *values_ptr,
-                const index_type numNNZ) -> PrecisionT {
+                const int64_t numNNZ) -> PrecisionT {
```

Comment:
yes, there are always `int64_t`. The reason why I set it as `int64_t` here is: 1, it's required by `cuSparse` backend; 2. It's also easy for our maintenance since it clearly shows that LGPU backend's SparseH is different.